app/src/utils/mappings.js

```diff
@@ -8,6 +8,8 @@ const extractColumns = (mappings, key) =>
 
 const META_FIELDS = ['_index', '_type', '_score', '_click_id'];
 
+const NUMBER_FIELDS = ['integer', 'float', 'long', 'double'];
```

Comment:
@ShahAnuj2610 minor thing, can we rename this to `NUMERIC_DATATYPES` and add 
remaining numeric types like `short`, `half_float`, etc.
https://www.elastic.co/guide/en/elasticsearch/reference/current/number.html